frontend/src/metabase/visualizations/components/Visualization/Visualization.tsx

```diff
@@ -513,51 +726,86 @@ class Visualization extends PureComponent {
               error={errorMessageOverride ?? error}
               icon={errorIcon}
               isSmall={small}
-              isDashboard={isDashboard}
+              isDashboard={!!isDashboard}
             />
           ) : genericError ? (
             <SmallGenericError bordered={false} />
           ) : loading ? (
-            <LoadingView expectedDuration={expectedDuration} isSlow={isSlow} />
+            <LoadingView
+              expectedDuration={expectedDuration}
+              isSlow={!!isSlow}
+            />
           ) : (
-            <div
-              data-card-key={getCardKey(series[0].card?.id)}
-              className={cx(CS.flex, CS.flexColumn, CS.flexFull)}
-            >
-              <CardVisualization
-                {...this.props}
```

Comment:
The removal of this line actually may introduce a lot of issues because some props are not described in `VisualizationProps` type in `visualization.ts`, for example it requires passing the `dispatch` or `mode`.

I'll try to find all such props

Reply:
cc @deniskaber